doc/classes/DirAccess.xml

```diff
@@ -253,7 +253,35 @@
 			<param index="0" name="path_a" type="String" />
 			<param index="1" name="path_b" type="String" />
 			<description>
-				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object. Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (e.g., identical copies of the file that are not symbolic links).
+				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object (taking into account case sensitivity, relative paths, symbolic links, and hard links). Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (i.e. identical copies of a file that are not symbolic links).
+				[codeblocks]
+				[gdscript]
+				var dir = DirAccess.open("/")
+
+				print(dir.is_equivalent("/sbin/md5sum", "/sBin/MD5Sum")) # true - on case insencetive filesystem
+
+				dir.create_link(ProjectSettings.globalize_path("res://icon.svg"), ProjectSettings.globalize_path("res://icon2.svg"))
+				print(dir.is_equivalent(ProjectSettings.globalize_path("res://icon.svg"), ProjectSettings.globalize_path("res://icon2.svg"))) # true - symlink
+				dir.remove(ProjectSettings.globalize_path("res://icon2.svg"))
+
+				dir.copy(ProjectSettings.globalize_path("res://icon.svg"), ProjectSettings.globalize_path("res://icon2.svg"))
+				print(dir.is_equivalent(ProjectSettings.globalize_path("res://icon.svg"), ProjectSettings.globalize_path("res://icon2.svg"))) # false - identical file copy
+				dir.remove(ProjectSettings.globalize_path("res://icon2.svg"))
+				[/gdscript]
+				[csharp]
+				[/csharp]
+				using var dir = DirAccess.Open("/");
+
+				GD.Print(dir.IsEquivalent("/sbin/md5sum", "/sBin/MD5Sum")); // true - on case insencetive filesystem
```

Comment:
```suggestion
				// Prints `true` on a case-insensitive filesystem, `false` otherwise.
				GD.Print(dir.IsEquivalent("/sbin/md5sum", "/sBin/MD5Sum"));
```